scripts/check_model.py

```diff
@@ -0,0 +1,57 @@
+"""
+Check for db indexes defined in mapping.py but missing in the database.
+Note: pass '-c /path/to/galaxy.yml' to use the database_connection set in galaxy.yml.
+Otherwise the default sqlite database will be used.
+"""
+import json
+import os
+import sys
+from collections import namedtuple
+
+sys.path.insert(1, os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, 'lib')))
+
+from sqlalchemy import create_engine, MetaData
+
+from galaxy.model import mapping
+from galaxy.model.orm.scripts import get_config
+
+Index = namedtuple('MissingIndex', 'table column_names')
```

Comment:
I think this type can be confused with https://docs.sqlalchemy.org/en/13/core/constraints.html#sqlalchemy.schema.Index . Also, the first argument of `namedtuple()` is usually the name of the new type as a string. So what about:
```suggestion
IndexTuple = namedtuple('IndexTuple', 'table column_names')
```

Reply:
Yes, definitely! Agreed on both. Will adjust the rest.